woocommerce/woocommerce-ios
# Shipping Labels: Update Networking model Shipping Label with new property commercialInvoiceURL

Part of #4597 

# Description
This PR updates the Networking layer with new property `commercialInvoiceURL` for Shipping Label model.

# Changes
- Added new property `commercialInvoiceURL` for ShippingLabel model in Networking layer.
- Updated the convert between Networking model and Storage model of ShippingLabel.
- Updated unit tests for new property `commercialInvoiceURL`.

# Testing
This PR touches only the underlying parts, so all unit tests need to pass. Parsing Shipping Label model should still work properly, i.e shipping label should be present in order details screen if created. Steps to test:
1. Make sure that your test store is eligible for creating shipping labels (has WCShip plugin installed, with shipping packages set up in WPAdmin > WooCommerce > Settings > Shipping > WooCommerce Shipping).
2. Create an order with the US being both origin and destination country (do not select military states) - to make sure that no customs form is required for the shipment.
3. Navigate to Orders tab in the app, select the new order, select Create Shipping Label and finish all the steps.
4. Navigate back to Order Details screen, notice that Shipping Label section is visible.
5. Create an international order (origin and destination countries are different) to require customs form for the shipment.
6. Repeat steps 3 and 4 for the new order. Notice that Shipping Label section should still be visible for the new shipping label.

Update release notes:
